ClosedXML/Excel/Ranges/XLRangeColumns.cs

```diff
@@ -48,23 +48,32 @@ IEnumerator IEnumerable.GetEnumerator()
 
         public IXLCells Cells()
         {
-            var cells = new XLCells(false, false);
+            var cells = new XLCells(false, XLCellsUsedOptions.Contents);
```

Comment:
```var cells = new XLCells(usedCellsOnly: true, options: options);``` does not look very nice

Reply:
Hmm, I got this message and thought it would not compile:
![image](https://user-images.githubusercontent.com/19576939/48250111-a09f2d80-e416-11e8-981f-21f89ffe6ad4.png)

But in fact, it succeeds.

OK, will add names to the arguments.